src/webapp/pages/dashboard/hooks/useDashboard.tsx

```diff
@@ -70,6 +84,7 @@ export function useDashboard() {
                 value={"somevalue"}
                 label={i18n.t("Status")}
             />
+            <MenuButton label={"New Data Quality"} items={menuButtonItems} />
```

Comment:
i18n for the string
```tsx
<MenuButton label={i18n.t("New Data Quality")} items={menuButtonItems} />
```

Reply:
In this case, translation was set in the original component, but should we translate in the instances?

//MenuButton.tsx

```
       <Button
                aria-controls="simple-menu"
                aria-haspopup="true"
                variant="contained"
                color="primary"
                onClick={onOpenMenu}
                endIcon={anchorEl ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
            >
                {i18n.t(label)}
            </Button>
```

Reply:
Yes, i18n needs to know in advance what's the string you want to translate (when running `yarn localize`) in order to generate them into *.po files. In this case we can remove i18n in the component.